example/gardener-local/gardenlet/values.yaml

```diff
@@ -36,6 +36,7 @@ config:
     CopyEtcdBackupsDuringControlPlaneMigration: true
     DefaultSeccompProfile: true
     CoreDNSQueryRewriting: true
+    FullNetworkPoliciesInRuntimeCluster: false
```

Comment:
What is the reason we don't enable this feature gate here?

Reply:
We use `networking-calico` in the local setup and it does not yet support the feature gate. I'll add a comment.